README.md

```diff
@@ -171,6 +170,40 @@ var userInfo = new RaygunIdentifierMessage("12345")
 
 Log.ForContext("CustomUserInfoProperty", userInfo, true).Error(new Exception("random error"), "other information");
 ```
+
+### onBeforeSend
+`type: Action<OnBeforeSendParameters>`
+
+`default: null`
+
+The field allows users to manipulate the crash report payload to be sent to Raygun.
+By default it is `null`, so you don't need to set it in the constructor. If the field is `null`, nothing happens; if an `Action<OnBeforeSendParameters>` is passed, it gets called just before the crash report payload gets serialized and sent to Raygun.
+The arguments to the action are of type `Struct OnBeforeSendArguments`; they are passed to the action when it is called and contain references to the following objects passed by the Raygun client object:
+```csharp
+//Abstracted away version of the struct to just show the fields
+struct OnBeforeSendArguments
+{
+    System.Exception Exception;
+    Mindscape.Raygun4Net.Messages.RaygunMessage RaygunMessage;
+}
+```
+
+The provided action can read and/or modify their fields accordingly to produce the desired effect.
```

Comment:
I'd change"field" to "property" in the couple of places it appears in this scentence